src/API/Google/Query/MerchantPriceBenchmarksQuery.php

```diff
@@ -0,0 +1,52 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query\MerchantQuery;
+
+/**
+ * Class MerchantPriceBenchmarksQuery
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\API\Google\Query
+ */
+class MerchantPriceBenchmarksQuery extends MerchantQuery {
+
+	/**
+	 * MerchantPriceBenchmarksQuery constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'PriceCompetitivenessProductView' );
+		$this->set_initial_columns();
+	}
+
+	/**
+	 * Filter the query by a list of product IDs.
+	 *
+	 * @param array $ids List of product IDs to filter by.
+	 *
+	 * @return $this
+	 */
+	public function filter( array $ids ): QueryInterface {
+		if ( ! empty( $ids ) ) {
+			$this->where( 'product_view.id', 'IN', $ids );
```

Comment:
Not sure that we really need to be able to filter by IDs, but even if so, the last two args in the `where()` method here look to be reversed:
```suggestion
			$this->where( 'product_view.id', $ids , 'IN' );
```

Same for the filter method in the `MerchantPriceSuggestionsQuery` object